src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const LOAD = 'bookstore/books/LOAD';
+const CREATE = 'bookstore/books/CREATE';
+const UPDATE = 'bookstore/books/UPDATE';
+const REMOVE = 'bookstore/books/REMOVE';
+
+export default function reducer(state = {}, action = {}) {
+  switch (action.type) {
+    default: return state;
+  }
+}
+
+export function loadBooks() {
+  return { type: LOAD };
+}
+
+export function createBook(book) {
+  return { type: CREATE, book };
+}
+
```

Comment:
- [ ] You should render a form with inputs for the title and the author of the book
- [ ] On the form add a button for submit the button wouldn't add the book *yet*, this is just a placeholder for now

Reply:
Done.

Reply:
- [ ] Well addressed this but you can remember to add `author` inputs as well